community/php7/APKBUILD

```diff
@@ -86,9 +86,10 @@ _build() {
 		--with-config-file-path=/etc/$pkgname \
 		--with-config-file-scan-dir=/etc/$pkgname/conf.d \
 		--disable-short-tags \
+		--enable-fileinfo=shared \
 		--with-openssl=shared --with-kerberos --with-system-ciphers \
-		--with-pcre-regex --with-pcre-dir=/usr \
-		--with-zlib=shared --with-zlib-dir=/usr \
+		--with-pcre-regex=/usr --with-pcre-dir \
+		--with-zlib=shared --with-zlib-dir \
```

Comment:
Why did you change `--with-zlib-dir=/usr` to `--with-zlib-dir`? Doesn’t it mean that bundled lib will be used instead of the system-provided?